server/models/development/bhima_data.sql

```diff
@@ -354,7 +355,7 @@ UNLOCK TABLES;
 
 LOCK TABLES `enterprise` WRITE;
 
-insert  into `enterprise`(`id`,`name`,`abbr`,`phone`,`email`,`location_id`,`logo`,`currency_id`,`po_box`) values (200,'Hopital Bon Berger','IMCK','243 81 604 0540','imckbusiness@yahoo.fr','bda70b4b-8143-47cf-a683-e4ea7ddd4cff','/assets/logos/tsh.jpg',2,'205');
+insert  into `enterprise`(`id`,`name`,`abbr`,`phone`,`email`,`location_id`,`logo`,`currency_id`,`po_box`) values (200,'Institut Medical Chretien de Kananga','IMCK','243 81 604 0540','imckbusiness@yahoo.fr','bda70b4b-8143-47cf-a683-e4ea7ddd4cff','/assets/logos/tsh.jpg',2,'205');

```

Comment:
Ce n'est pas de Kasai?  Also, I think this is probably better to include in `synt.sql`.